rootfs/api/serializers.py

```diff
@@ -474,3 +475,18 @@ class Meta:
         """Metadata options for a :class:`AppSettingsSerializer`."""
         model = models.AppSettings
         fields = '__all__'
+
+    def validate_whitelist(self, data):
```

Comment:
even they use the ipaddress module of the python which i am using to validate and i need to handle it the same way as each value in the array can be an ip or cidr so saving as string is the best way i can think of.